vrk-kpa/suomidigi
# Suopa_x_community_list

To test: 

- Make fresh
- Make new "Landing-sivu" from admin > content(sisältö) > add content > Landing-sivu. 
- Add a Paragraph > add view (Lisää näkymä) to the Landing-sivu. Choose a view "Communities" and publish.
- Make a couple of Community views from Content > Add content > Community. 
- Add a title (otsikko), choose a service (palvelu) and add a description (kuvaus) and add some link. 
- When you have added couple of Community views, open the Landing-sivu you created. 
- Compare the Landing-sivu to this visual layout: https://xd.adobe.com/spec/ba71d258-eba5-4a06-57b8-e3b89749d908-cc40/screen/75431cca-7934-4a1e-a9cb-b1533b68b5ff/Suomidigi-ryhmat-koonti/
- Check that they look the same

@khalima
